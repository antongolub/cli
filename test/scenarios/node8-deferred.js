'use strict'
require('../..')(main)

let foo

async function main () {
  console.log('started')
  console.log(foo)
  console.log('finished')
}

foo = 23