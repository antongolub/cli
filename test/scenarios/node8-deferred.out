started
23
finished
0
